Jupyter/VCS_Principles/VCS_Principles.html

```diff
@@ -3,8 +3,6 @@
 title: VCS_Principles Tutorial
 ---
 
-# VCS_Principles Tutorial
```

Comment:
we need the download link.

Reply:
The text I removed that looks like a download link for the notebook, was not an active link. It appears to be markdown code that did not get converted properly to html so it should be removed.
I did not delete the true link for the notebook which is included in the following html code:
<li><a href="VCS_Principles.ipynb">Download the Jupyter notebook</a></li>
Once the question about the example listed in the getting_started.md file has been resolved, @doutriaux1 could you review this pull request again in light of this explanation?